primefaces/src/main/resources/META-INF/primefaces.taglib.xml

```diff
@@ -11173,6 +11173,14 @@
             <required>false</required>
             <type>java.lang.Boolean</type>
         </attribute>
+         <attribute>
+            <description>
+                <![CDATA[If true, components which autoUpdate="true" will not be updated for this request. If not specified, or the value is false, no such indication is made.]]>
```

Comment:
same here; its only p:autoUpdate anymore

Reply:
i will fix all the taglib docs as this was copy and pasted!